### PR TITLE
docs: add redirects file for netlify

### PR DESCRIPTION
**Pull request by debs-obrien**
In Netlify you can create a _redirects file with the rules as I have put and it is quite an easy way of making netlify work for page refreshes in spa mode

### CODE REVIEW

**Comment by manniL** [comment]
Suggestion: Incorporate https://github.com/nuxt-community/netlify-files-module in the docs too :relaxed:

File: en/faq/netlify-deployment.md [hunk 1]
```diff
@@ -35,15 +35,24 @@ Press the _"New site from Git"_ button on the Netlify dashboard. Authenticate wi
 1. __Build command:__ `npm run build`
 1. __Publish directory:__ `dist`
 
-For a single page app there is a problem with refresh as by default on netlify the site redirects to *"404 not found"* this can be [prevented](https://www.netlify.com/docs/redirects/#rewrites-and-proxying) by configuring redirects. There is a module for that [nuxt-netlify](https://www.bazzite.com/docs/nuxt-netlify) which helps to configure both headers and redirects of the SPA .
+For a single page app there is a problem with refresh as by default on netlify the site redirects to *"404 not found"*. For any pages that are not generated they will fallback to SPA mode and then if you refresh or share that link you will get Netlify's 404 page.
 
-Alternativaly you can create a `_redirects` file (no file extension just _redirects) in the `static` folder with the following rules:
+The easiest way to fix this is by adding a generate property in your `nuxt.config` and setting `fallback: true`. Then it will fallback to the 200.html when in SPA mode.
```
Comment (manniL) [inline en/faq/netlify-deployment.md]: Maybe we could link to https://nuxtjs.org/api/configuration-generate#fallback here too :relaxed:

File: en/faq/netlify-deployment.md [hunk 2]
```diff
@@ -35,15 +35,24 @@ Press the _"New site from Git"_ button on the Netlify dashboard. Authenticate wi
...
-/*    /200.html   200
-/*    /404.html   404
+export default {
+  generate: {
+    fallback: false
+  }
+}
 ```
 
+If however you want to automatically apply headers and redirects of the SPA then there is a module for that:
```
Comment (manniL) [inline en/faq/netlify-deployment.md]: Could you emphasize that the module is especially useful when you have *custom* headers/redirects (in a `_headers` or`_redirects` file)? :)

File: en/faq/netlify-deployment.md (hunk 1 above)
Comment (debs-obrien) [reply en/faq/netlify-deployment.md]: done

File: en/faq/netlify-deployment.md (hunk 2 above)
Comment (debs-obrien) [reply en/faq/netlify-deployment.md]: done

**Comment by manniL** [comment]
Many thanks! :relaxed:

**Pull request merged**